openvidu-browser/src/OpenVidu/OpenVidu.ts

```diff
@@ -946,4 +947,4 @@ export class OpenVidu {
     }
   }
 
-}
\ No newline at end of file
+}
```

Comment:
Please, remove this new line char
```suggestion
```

Reply:
Done, seems to be a bug in the GiHib online editor....

Reply:
Last closing bracket has been removed. It was only supposed to be the last new line character, not last bracket. If you don't mind I'll commit the change myself to integrate it right now on the master branch, as new release is coming soon.

Reply:
ok, perfext, thx!